.organizationmap

```diff
@@ -1141,46 +1335,58 @@ Unaffiliated <no@organization.net> Tim Bishop <tim@bishnet.net>
 Unaffiliated <no@organization.net> Tim Freund <tim@freunds.net>
 Unaffiliated <no@organization.net> Timofey Titovets <nefelim4ag@gmail.com>
 Unaffiliated <no@organization.net> Tina Kallio <tina.kallio@gmail.com>
+Unaffiliated <no@organization.net> Tobias Bossert <tobias.bossert@fastpath.ch>
 Unaffiliated <no@organization.net> Tobias Gall <tobias.gall@mailbox.org>
 Unaffiliated <no@organization.net> Tobias Suckow <tobias@suckow.biz>
+Unaffiliated <no@organization.net> Tobias Urdin <tobias.urdin@binero.se>
 Unaffiliated <no@organization.net> Tomasz Setkowski <tom@ai-traders.com>
 Unaffiliated <no@organization.net> TommyLike <tommylikehu@gmail.com>
 Unaffiliated <no@organization.net> Tomáš Kukrál <tom@6shore.net>
+Unaffiliated <no@organization.net> Tongliang Deng <dengtongliang@gmail.com>
 Unaffiliated <no@organization.net> Tsung-Ju Lii <usefulalgorithm@gmail.com>
 Unaffiliated <no@organization.net> Tyler Zeqing Qi <aqize@126.com>
 Unaffiliated <no@organization.net> Uday Mullangi <udaymjl@gmail.com>
+Unaffiliated <no@organization.net> Uli Fahrer <github@uli-fahrer.de>
+Unaffiliated <no@organization.net> Umanga Chapagain <chapagainumanga@gmail.com>
```

Comment:
I think this should linked to Red Hat. cc @umangachapagain

Reply:
@umangachapagain is there a redhat email address you'd like to be associated with your contributions? or just the gmail one?

Reply:
Gmail one is fine.